demos/blast/blast_in_example.cpp

```diff
@@ -0,0 +1,43 @@
+#include <iostream>
+#include <seqan/blast.h>
+
+using namespace seqan;
+
+int main()
+{
+    typedef Align<String<AminoAcid>, ArrayGaps> TAlign;
+    typedef BlastMatch<CharString, CharString, uint32_t, TAlign> TBlastMatch;
```

Comment:
Yep we have them with C++11 (see http://www.cplusplus.com/reference/cstdint/). But please change it back for now.